e2e/main.e2e.spec.ts

```diff
@@ -57,38 +57,10 @@ test.describe.serial('Main App Test', () => {
     await expect(navPage.progressBar).toBeHidden();
   });
 
-  test('should navigate to Kubernetes Settings and check elements', async() => {
-    const navPage = new NavPage(page);
-    const k8sPage = await navPage.navigateTo('K8s');
-
-    if (!os.platform().startsWith('win')) {
-      await expect(k8sPage.memorySlider).toBeVisible();
-      await expect(k8sPage.cpuSlider).toBeVisible();
-    } else {
-      // On Windows memory slider and cpu should be hidden
-      await expect(k8sPage.memorySlider).toBeHidden();
-      await expect(k8sPage.memorySlider).toBeHidden();
-    }
-
-    await expect(navPage.mainTitle).toHaveText('Kubernetes Settings');
-    await expect(k8sPage.port).toBeVisible();
-    await expect(k8sPage.resetButton).toBeVisible();
-    await expect(k8sPage.engineRuntime).toBeVisible();
-    await expect(k8sPage.enableKubernetes).toBeVisible();
-  });
-
   /**
    * Checking WSL and Port Forwarding - Windows Only
```

Comment:
We're no longer checking WSL here; the comment should be updated.

Reply:
We actually port forward on all platforms today.. This has been fixed.